.github/workflows/int-test-workflow.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - develop
       - master
+      - CSPL-1309-Cluster-Manager-CRD
```

Comment:
Please ignore, this is only for testing purposes. It will be removed after reviews.